diff --git a/external.bzl b/external.bzl
index 20b6cbb..23a3036 100644
--- a/external.bzl
+++ b/external.bzl
@@ -325,7 +325,7 @@ def _java_dependencies():
             "org.apache.tomcat:tomcat-annotations-api:9.0.45",
             "junit:junit:4.13.2",
             "org.checkerframework:checker-qual:3.12.0",
-            "org.ow2.asm:asm:9.1",
+            "org.ow2.asm:asm:9.7.1",
         ],
         repositories = [
             "https://jcenter.bintray.com",
